app/assets/stylesheets/_orders.scss

```diff
@@ -3,3 +3,54 @@
   margin:0 auto;
   margin-bottom: 30px;
 }
+
+.admin-order-title{
+  text-align:center;
+  margin-bottom:30px;
+}
+
+.admin-order-show{
+  width:70%;
+  margin:0 auto;
+  border:1px solid white;
+  background:rgba(255, 255, 255, 0.91);
+
+  tfoot{
+    td{
+      text-align:center;
+    }
+  }
+}
+
+.admin-order-list{
+  margin:0 auto;
```

Comment:
Properties should be ordered background, border, margin, margin-top, text-align, width<br>Colon after property should be followed by one space